util/find_missing_translations.ts

```diff
@@ -162,7 +162,7 @@ const findMissingTranslations = async (filter: string, locale: Lang): Promise<vo
         let str = file;
         if (line)
           str += `:${line}`;
-        if (label)
+        if (label !== undefined && label.length > 0)
```

Comment:
```suggestion
        if (label !== undefined)
```

I don't think any callers of this special case empty string, so no need to handle it specially.